include/avif/avif.h

```diff
@@ -810,6 +814,12 @@ typedef struct avifDecoder
     avifBool ignoreExif;
     avifBool ignoreXMP;
 
+    // This represents the maximum size of a image (in pixel count) that libavif and the underlying
+    // AV1 decoder should attempt to decode. It defaults to AVIF_DEFAULT_MAX_IMAGE_SIZE, and can be
+    // set to a smaller value. The value 0 is reserved.
+    // Note: Only some underlying AV1 codecs support a configurable size limit (such as dav1d).
+    uint32_t imageSizeLimit;
```

Comment:
Joe: Any preference for whether the `imageSizeLimit` member should be declared before or after the `imageCountLimit` member?

Reply:
Either order here is acceptable.